examples/e-commerce/src/App.vue

```diff
@@ -515,8 +514,8 @@ export default {
     formatNumber,
     toValue(value, range) {
       return [
-        value.min !== null ? value.min : range.min,
-        value.max !== null ? value.max : range.max,
+        value.min != null ? value.min : range.min,
+        value.max != null ? value.max : range.max,
```

Comment:
Would not be better to test for `value.min !== null && value.min !== undefined` or `typeof value.min === 'number'` (if this is the correct condition) here?